doofinder-for-woocommerce/assets/js/doofinder-integration-klaviyo.js

```diff
@@ -0,0 +1,38 @@
+/** START INTEGRATION WITH KLAVIYO **/
+window.addEventListener('load', async (event) => {
+  if (typeof klaviyo !== 'undefined' && await klaviyo.isIdentified() === false && typeof klUser !== 'undefined' && klUser.current_user_email !== "") {
```

Comment:
Unlike PrestaShop, WordPress enforces Yoda Conditions (at least in PHP)
, but it's better to keep consistency between the shared rules:
```suggestion
  if ('undefined' !== typeof klaviyo && false === await klaviyo.isIdentified() && 'undefined' !== typeof klUser && "" !== klUser.current_user_email) {
```